axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConfiguration.java

```diff
@@ -77,26 +78,27 @@
      */
     private boolean sslEnabled;
 
+
     /**
-     * Initial number of permits send for message streams (events, commands, queries)
+     * Default flow control settings.
      */
-    private Integer initialNrOfPermits = 5000;
```

Comment:
Changing these properties (fields as well as getters and setters) will result in a change how the Configuration bean is constructed from the properties file. While keeping the getters and setters will ensure the old configuration style keeps working, the generated documentation file will no longer contain the old properties.
I suggest keeping these fields and adding 3 getters that reflect the flow control settings for the message types. Any setting in the specific flow control settings that is null can then be replaced with the defaults.

Reply:
ok. will change that. although i tested it local with a property file

Reply:
@abuijze did your suggested modifications. Is there a way to check the generated documentation file ?